SmurfTestingScripts/README.md

```diff
@@ -38,41 +58,64 @@ because they demand human-approve.
 ### Scenarios
 
 #### keys:
-- setup : set of commands, that are run once on the beginning
-- between_runs : set of commands, that are run in between each test scenario
-- cleanup : set of commands, that are run once in the end of tests
-- scenarios : set of testing scenarios containing:
-  - name : test name
-  - timeout : time in seconds after which the run is terminated (SIGTERM). 10 seconds after SIGTERM, SIGKILL is sent 
-  - arguments : set of program arguments. Switch without value has to be an empty string!
-  
+- `setup` : list of commands, that are run once at the beginning
+- `between_runs` : list of commands, that are run in between each test scenario
+- `cleanup` : list of commands, that are run once at the end of all tests
+- `default_executable` (optional) : path to the tested executable; the --executable argument has priority over this
+- `scenarios` : set of testing scenarios containing:
+  - `name` : test name; must be unique
+  - `timeout_s` (optional) : time in seconds (can be decimal) after which the run is terminated (SIGTERM). 10 seconds after SIGTERM, SIGKILL is sent 
+  - `arguments` (optional) : list of program arguments for the tested executable
+  - `actions` (optional) : list of commands, that are run in parallel with the tested executable. If any of the commands exit with a non 0 return code, the test fails. Make sure actions are written in a way, where no action is running after the tested executable ends (whis may lead to commands running during clean-up steps or other tests)
```

Comment:
Please note that the STATE_SMURF_ENV[] + env file can be used in all of these fields. It is not clear now. 
You can try it to make it clear with a standalone paragraph pls.

Reply:
added mention about STATE_SMURF_ENV